issues.md

```diff
@@ -171,60 +165,41 @@ We need an assigned triager to regularly triage bugs coming in on each MDN conte
 - WebAPI — Ruth John
 - WebExt — Caitlin/WebExt team
 
-### Set initial labels
-
-As soon as a new issue is filed, the MDN core team, and anyone else who wishes to help, will add the following labels to that issue:
-
-- `needs-triage` — signals that this issue needs a proper triage, to get it ready to work on (this should happen automatically).
-- `Content:<area>` — specifies the content topic this issue relates to, e.g. `Content:HTML` or `Content:CSS`. This needed for the triagers to be able to find the issues in their specific areas.
-- `l10n-fr`, `l10n-zh`, `l10n-ja` — specifies that the issue filed concerns an active non-en-US locale. The teams for those locales will pick these issues up and triage them.
-
-### Set aside triage time
-
-Triagers don't need to be actively triaging bugs all the time. Instead, they should set out a 30-minute slot in which to triage the bugs in their area of responsibility, each week.
-
-This doesn't have to be done as part of a synchronous meeting, or even at the same time as everyone else, but it should be done regularly, say once per week, to make sure that the backlog of untriaged bugs doesn't get too high.
+### Issue-specific triaging tasks
 
-## Triage process for each issue
+These are the guidelines to follow while triaging each issue.
 
-### Checklist to determine if we have enough information
+#### Review the issue to determine the completeness of information
 
-For each bug, run through the following checklist to make sure the issue contains enough information for someone to start working on the bug.
+Review each issue against the following checklist to ensure that the issue contains the described information for someone to start working on the bug:
 
-Does the issue contain:
-
-- The MDN URL where the problem has been found.
-- The URL of any example page or repo related to the bug, if appropriate.
-- The specific heading on the MDN page where the problem can be found (if needed to find it).
+- MDN URL where the problem has been found.
+- The specific heading or section on the MDN page where the problem can be found.
+- URL of an example page or repository related to the bug, if appropriate.
```

Comment:
Are we repeating the need for a URL in this sentence?

Reply:
Fixed. TBH, I am not sure what the second one is referring to.